src/backends/backend.js

```diff
@@ -97,6 +98,14 @@ class Backend {
         {
           entries: entries.map(this.entryWithFormat(collection)),
         }
+      ))
+      // If this collection has a "filter" property, filter entries accordingly
+      .then(loadedCollection => (
+        {
+          entries: loadedCollection.entries.filter(
+            entry => (!collectionFilter || entry.data[collectionFilter.get('field')] === collectionFilter.get('value'))
```

Comment:
This is going to loop over the full collection even when there is no filter, can we not do that?

I would expect doing something like `collectionFilter ? loadedCollection.entries.filter(...) : loadedCollection.entries`.
It would also be great it was a separated function if this becomes hard to read, something like:

```
.then(loadedCollection => {entries: filterEntries(loadedCollection, collectionFilter)})
```